src/CounterInterceptor/README.md

```diff
@@ -0,0 +1,28 @@
+# dotnet-counter-interceptor
+The sample demonstrates: 
+- the use of a Worker Workflow Interceptor that counts the number of Workflow Executions, Child Workflow Executions, and Activity Executions and the number of Signals and Queries. It is based
+off of the [Java sample](https://github.com/temporalio/samples-java/tree/main) located [here](https://github.com/temporalio/samples-java/tree/main/core/src/main/java/io/temporal/samples/countinterceptor)
+- the use of a Client Workflow Interceptor that counts the number of Workflow Executions and the number of Signals and Queries.
+
+## Start local Temporal Server
+```bash
+# run only once
+temporal server start-dev
+```
+
+## Run Worker Locally
```

Comment:
This is more than the worker, it's the worker and the client calls mixed into one. Many samples split these, but don't have to here, but it's not just "worker" as the title suggests.

Reply:
Bump, this was never addressed

Reply:
Bump again, this is still not addressed

Reply:
Fixed.